frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -133,19 +133,19 @@ pub mod pallet {
     /// Registered developer accounts points to coresponding contract
     #[pallet::storage]
     #[pallet::getter(fn registered_contract)]
-    pub(crate) type RegisteredDevelopers<T: Config> =
```

Comment:
This would mean that these items can also be modified from the outside - that doesn't seem good.

Can the derived methods be used instead? They should be public. E.g. `registered_contract` method for this particular storage type.

Reply:
I tried and scope for the derived getters is the same. I also checked other project and it is clear that functions called by precompiles are public